frontends/api/src/test-utils/factories/userLists.ts

```diff
@@ -12,6 +13,8 @@ const userList: Factory<UserList> = (overrides = {}) => {
   const list: UserList = {
     id: faker.helpers.unique(faker.datatype.number),
     title: faker.helpers.unique(faker.lorem.words),
+    description: faker.helpers.unique(faker.lorem.paragraph),
+    privacy_level: PrivacyLevelEnum.Private,
```

Comment:
If you want, you can do `faker.helpers.arrayElement(Object.values(PrivacyLevelEnum))`.

But this is fine too